### PR TITLE
Bug/ux 140 status update indicator

### PR DESCRIPTION
**Pull request by gnklausner**

### CODE REVIEW

**Comment by chriskacerguis** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54d4fa726956803255002db9/compare/bug/ux-140-status-update-indicator">analyzed this pull request</a>.

**Comment by gnklausner** [comment]
Fixed merge conflicts

**Comment by stackus** [comment]
Could you make sure your IDE is set to use two spaces for indenting?

File: assets/js/app/admin/products/product_form_controller.js [hunk 1]
```diff
@@ -17,24 +17,77 @@ function ProductFormController($state, FlashesService) {
     this.product = parent.product;
   };
 
-  function onSuccess() {
+  this.onSuccess = function(actionType) {
+      if (actionType === 'create') {
+          FlashesService.add({
+              timeout: true,
+              type: 'success',
+              message: 'Product was successfully added.'
```
Comment (chriskacerguis) [inline assets/js/app/admin/products/product_form_controller.js]: Can you check to see if the API is returning a message?  It should be and the API should just display things.

Comment (gnklausner) [inline assets/js/app/admin/products/product_form_controller.js]: Can you elaborate on what the message may look like? All I'm finding is '201 created' and JSON that describes the product (no message).

Comment (chriskacerguis) [inline assets/js/app/admin/products/product_form_controller.js]: Hmm...it was doing that, but I guess it is not.  All is good.  I'll merge.

**Comment by gnklausner** [comment]
Closes #140

**Pull request merged**